lib/config/presets/internal/abandonments.ts

```diff
@@ -0,0 +1,38 @@
+import dataFiles from '../../../data-files.generated';
+import type { PackageRule } from '../../types';
+import type { Preset } from '../types';
+
+/* v8 ignore start: not testable */
+function loadAbandonmentPresets(): Record<string, Preset> {
+  const data: Record<string, Record<string, string>> = JSON.parse(
+    dataFiles.get('data/abandonment-config.json')!,
+  );
+  delete data.$schema;
+
+  const packageRules: PackageRule[] = [
+    {
+      matchPackageNames: ['*'],
+      abandonmentThreshold: '1 year',
+    },
```

Comment:
Maybe a higher one for npm `@types/*`? @viceice wdyt

Reply:
sounds good